src/main/resources/schema/ans/0.8.1/traits/trait_content_restrictions.json

```diff
@@ -8,6 +8,27 @@
     "content_code":{
       "type": "string",
       "description": "The content restriction code/level/flag associated with the ANS object"
+    },
+    "embargo": {
+      "description": "Embargo configuration to enforce publishing restrictions",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "active": {
+          "type": "boolean",
+          "description": "The boolean flag to indicate if the embargo is active or not"
```

Comment:
Can we change this description to indicate that it is an "enable" toggle and not a calculated field? Something like "If this field is false, ignore the embargo." I'd even be in favor of renaming to `disabled` (and not require it), but updating the description would go a long way.